src/redux/reducer.js

```diff
@@ -31,8 +34,11 @@ const paramsInitialState = {
 
 const initialState = {
     computedLanguage: getLocalStorageComputedLanguage(),
+    currentChildren: null,
```

Comment:
What is the difference between  currentChildren and selectedDirectory ?

Reply:
currentChildren are the children of a selectedDirectory (studies objects etc...)